app/containers/BuyRam/index.js

```diff
@@ -34,8 +34,31 @@ const withConnect = connect(
 const withReducer = injectReducer({ key: 'BuyRam', reducer });
 const withSaga = injectSaga({ key: 'BuyRam', saga });
 
-export default compose(
+const enhance = compose(
   withReducer,
   withSaga,
-  withConnect
-)(BuyRamForm);
+  withConnect,
+  withStateHandlers(
```

Comment:
using `recompose` and their awesome HOCs to handle all the radio logic.

Using `mapProps` to combine logic within a single `unit` prop, much cleaner when dealing with it in the presentational components